src/wallet/spend.cpp

```diff
@@ -105,6 +105,49 @@ void CoinsResult::clear()
     other.clear();
 }
 
+void CoinsResult::erase(std::set<COutPoint>& preset_coins)
```

Comment:
In a0abea26692e63e9205ec68bdbc04cfad8c95a3d "refactor: add new helper methods"

nit: We prefer UpperCamelCase for function names.